std/json.d

```diff
@@ -1719,3 +1727,37 @@ pure nothrow @safe unittest // issue 15884
     Test!dchar();
 }
 
+@safe unittest // issue 15885
+{
+    static bool test(const double num0)
+    {
+        import std.math : feqrel;
+
+        const json0 = JSONValue(num0);
+        const num1 = to!double(toJSON(json0));
+        const json1 = JSONValue(num1);
+        const num2 = to!double(toJSON(json1));
+        import std.stdio;
+        writefln("%.18g : %.18g : %.18g : %s", num0, num1, num2, feqrel(num2, num0));
+
+        version(Windows)
```

Comment:
What 's happening on Windows ? If the failure is related to DigitalMars snn.lib ? Is so then the version should be changed to `CRuntime_DigitalMars` because MS C standard lib might not have the issue.